### PR TITLE
[USING] rework Restore Electrum

### PR DESCRIPTION
**Pull request by MaxHillebrand**
This ready for review branch does a rework of the chapter on restoring wasabi to electrum. Mainly layout changes, and minor nits.

### CODE REVIEW

File: docs/using-wasabi/RestoreElectrum.md [hunk 1]
```diff
@@ -1,107 +1,89 @@
...
 }
 ---
 
-# Restoring Wasabi Wallet to Electrum
+# Restoring Wasabi Wallet in Electrum
 
-:::danger Bad for your privacy!
-If you do this, you send all your Wasabi addresses to 3rd party Electrum servers, losing the anonymity against those entities, so you must make a judgement call by being aware of this.
+:::danger Potential privacy leak!
+If you do not run a full node and your own Electrum server, you will leak all your addresses to 3rd party servers, losing anonymity against those entities, so you must make a judgement call by being aware of this.
```
Comment (RiccardoMasutti) [inline docs/using-wasabi/RestoreElectrum.md:11]:
```suggestion
If you do not run a full node and your own Electrum server, you will leak all your addresses to random third party servers, losing anonymity against those entities, so you must make a judgement call by being aware of this.
```

File: docs/using-wasabi/RestoreElectrum.md [hunk 2]
```diff
@@ -1,107 +1,89 @@
...
-4. Choose `I already have a seed` or `Use a master key`
+4. Choose `I already have a seed` or `Use a master key`.
 
-5. Type your seed or paste your master private key
+5. Type your seed or paste your master private key.
 
-:::danger You are at risk!
-By typing the seed or pasting the master private key on a hot or compromised device, you risk losing your funds.
-:::
+	:::tip Get your master private key inside Wasabi:
```
Comment (RiccardoMasutti) [inline docs/using-wasabi/RestoreElectrum.md:34]:
```suggestion
:::tip Get your master private key inside Wasabi:
```

File: docs/using-wasabi/RestoreElectrum.md [hunk 3]
```diff
@@ -1,107 +1,89 @@
...
+	- Go to `Advanced` tab -> `Wallet Info`.
```
Comment (RiccardoMasutti) [inline docs/using-wasabi/RestoreElectrum.md:35]:
```suggestion
- Go to `Advanced` tab -> `Wallet Info`.
```

File: docs/using-wasabi/RestoreElectrum.md [hunk 4]
```diff
@@ -1,107 +1,89 @@
...
+	- Copy your `Extended Account zprv`
```
Comment (RiccardoMasutti) [inline docs/using-wasabi/RestoreElectrum.md:36]:
```suggestion
- Copy your `Extended Account zprv`
```

File: docs/using-wasabi/RestoreElectrum.md [hunk 5]
```diff
@@ -1,107 +1,89 @@
...
+	:::
```
Comment (RiccardoMasutti) [inline docs/using-wasabi/RestoreElectrum.md:37]:
```suggestion
:::
```

File: docs/using-wasabi/RestoreElectrum.md [hunk 6]
```diff
@@ -1,107 +1,89 @@
...
 
-:::tip
-Steps to find your master private key inside Wasabi:
-- Go to `Advanced` tab -> `Wallet Info`.
-- Copy your `Extended Account zprv`
-:::
-
-The following steps are only necessary if you have previously chosen the option `I already have a seed`.
+	The following steps are only necessary if you have previously chosen the option `I already have a seed`.
```
Comment (RiccardoMasutti) [inline docs/using-wasabi/RestoreElectrum.md:39]:
```suggestion
The following steps are only necessary if you have previously chosen the option `I already have a seed`.
```

File: docs/using-wasabi/RestoreElectrum.md [hunk 7]
```diff
@@ -1,107 +1,89 @@
...
-
-8. Open Electrum's `Console` and type the following commands:
+8. Increase the gap limits by opening Electrum's `Console` and executing the following commands:
 
-```
-wallet.change_gap_limit(100)
-wallet.gap_limit_for_change = 100
-wallet.synchronize()
-```
+	```
```
Comment (RiccardoMasutti) [inline docs/using-wasabi/RestoreElectrum.md:47]:
```suggestion
```
```

File: docs/using-wasabi/RestoreElectrum.md [hunk 8]
```diff
@@ -1,107 +1,89 @@
...
+	wallet.change_gap_limit(100)
```
Comment (RiccardoMasutti) [inline docs/using-wasabi/RestoreElectrum.md:48]:
```suggestion
wallet.change_gap_limit(100)
```

File: docs/using-wasabi/RestoreElectrum.md [hunk 9]
```diff
@@ -1,107 +1,89 @@
...
+	wallet.gap_limit_for_change = 100
```
Comment (RiccardoMasutti) [inline docs/using-wasabi/RestoreElectrum.md:49]:
```suggestion
wallet.gap_limit_for_change = 100
```

File: docs/using-wasabi/RestoreElectrum.md [hunk 10]
```diff
@@ -1,107 +1,89 @@
...
+	wallet.synchronize()
```
Comment (RiccardoMasutti) [inline docs/using-wasabi/RestoreElectrum.md:50]:
```suggestion
wallet.synchronize()
```

File: docs/using-wasabi/RestoreElectrum.md [hunk 11]
```diff
@@ -1,107 +1,89 @@
...
+	```
```
Comment (RiccardoMasutti) [inline docs/using-wasabi/RestoreElectrum.md:51]:
```suggestion
```
```

File: docs/using-wasabi/RestoreElectrum.md [hunk 12]
```diff
@@ -1,107 +1,89 @@
...
-If your balance is not correct maybe it is necessary to change the `gap_limit` or the `gap_limit_for_change` to 100 or even more if needed.
-
-5. Open Electrum's `Console` and type the following commands:
-
-```
-wallet.change_gap_limit(100)
-wallet.gap_limit_for_change = 100
-wallet.synchronize()
-```
+	```
```
Comment (RiccardoMasutti) [inline docs/using-wasabi/RestoreElectrum.md:59]:
```suggestion
```
```

File: docs/using-wasabi/RestoreElectrum.md [hunk 13]
```diff
@@ -1,107 +1,89 @@
...
+	{
```
Comment (RiccardoMasutti) [inline docs/using-wasabi/RestoreElectrum.md:60]:
```suggestion
{
```

File: docs/using-wasabi/RestoreElectrum.md [hunk 14]
```diff
@@ -1,107 +1,89 @@
...
+	    "keystore": {
```
Comment (RiccardoMasutti) [inline docs/using-wasabi/RestoreElectrum.md:61]:
```suggestion
	"keystore": {
```

File: docs/using-wasabi/RestoreElectrum.md [hunk 15]
```diff
@@ -1,107 +1,89 @@
...
+	        "type": "bip32",
```
Comment (RiccardoMasutti) [inline docs/using-wasabi/RestoreElectrum.md:62]:
```suggestion
	"type": "bip32",
```

File: docs/using-wasabi/RestoreElectrum.md [hunk 16]
```diff
@@ -1,107 +1,89 @@
...
+	        "xprv": "vprv9DMUxX4ShgxMLku9TWoPTTt3ZKYhAHCuhULnR9Qmkv1naNQ4g1HfdURhfq65aJKk7zsmnoXbVryeLwtkj9LhuQoKFD5Fyus9kkiwA1S2pEU",
```
Comment (RiccardoMasutti) [inline docs/using-wasabi/RestoreElectrum.md:63]:
```suggestion
	"xprv": "vprv9DMUxX4ShgxMLku9TWoPTTt3ZKYhAHCuhULnR9Qmkv1naNQ4g1HfdURhfq65aJKk7zsmnoXbVryeLwtkj9LhuQoKFD5Fyus9kkiwA1S2pEU",
```

File: docs/using-wasabi/RestoreElectrum.md [hunk 17]
```diff
@@ -1,107 +1,89 @@
...
+	        "xpub": "vpub5ZPNxAuehdRrMpX7vQSQzAshpziQUZHnDkzPBZJ4GFbhn8mVxoyxJzCGq482a2BDHvfnMfX1u4hdX1XfS7ZdHJXffydLPHjCXBY86sG6tE1"
```
Comment (RiccardoMasutti) [inline docs/using-wasabi/RestoreElectrum.md:64]:
```suggestion
	"xpub": "vpub5ZPNxAuehdRrMpX7vQSQzAshpziQUZHnDkzPBZJ4GFbhn8mVxoyxJzCGq482a2BDHvfnMfX1u4hdX1XfS7ZdHJXffydLPHjCXBY86sG6tE1"
```

File: docs/using-wasabi/RestoreElectrum.md [hunk 18]
```diff
@@ -1,107 +1,89 @@
...
+	    },
```
Comment (RiccardoMasutti) [inline docs/using-wasabi/RestoreElectrum.md:65]:
```suggestion
	},
```

File: docs/using-wasabi/RestoreElectrum.md [hunk 19]
```diff
@@ -1,107 +1,89 @@
...
+	    "seed_type": "bip39",
```
Comment (RiccardoMasutti) [inline docs/using-wasabi/RestoreElectrum.md:66]:
```suggestion
	"seed_type": "bip39",
```

File: docs/using-wasabi/RestoreElectrum.md [hunk 20]
```diff
@@ -1,107 +1,89 @@
...
+	    "seed_version": 18,
```
Comment (RiccardoMasutti) [inline docs/using-wasabi/RestoreElectrum.md:67]:
```suggestion
	"seed_version": 18,
```

File: docs/using-wasabi/RestoreElectrum.md [hunk 21]
```diff
@@ -1,107 +1,89 @@
...
+	    "use_encryption": false,
```
Comment (RiccardoMasutti) [inline docs/using-wasabi/RestoreElectrum.md:68]:
```suggestion
	"use_encryption": false,
```

File: docs/using-wasabi/RestoreElectrum.md [hunk 22]
```diff
@@ -1,107 +1,89 @@
...
+	    "wallet_type": "standard"
```
Comment (RiccardoMasutti) [inline docs/using-wasabi/RestoreElectrum.md:69]:
```suggestion
	"wallet_type": "standard"
```

File: docs/using-wasabi/RestoreElectrum.md [hunk 23]
```diff
@@ -1,107 +1,89 @@
...
+	}
+	```
+
+3. Replace `xprv` and `xpub` sections with your Wasabi wallet's `Extended Account zprv` and `Extended Account zpub`.
+
+	:::tip Get your master private and public keys inside Wasabi:
```
Comment (RiccardoMasutti) [inline docs/using-wasabi/RestoreElectrum.md:75]:
```suggestion
:::tip Get your master private and public keys inside Wasabi:
```

File: docs/using-wasabi/RestoreElectrum.md [hunk 24]
```diff
@@ -1,107 +1,89 @@
...
+	- Go to `Advanced` tab -> `Wallet Info`.
```
Comment (RiccardoMasutti) [inline docs/using-wasabi/RestoreElectrum.md:76]:
```suggestion
- Go to `Advanced` tab -> `Wallet Info`.
```

File: docs/using-wasabi/RestoreElectrum.md [hunk 25]
```diff
@@ -1,107 +1,89 @@
...
+	- Copy your `Extended Account zprv`
```
Comment (RiccardoMasutti) [inline docs/using-wasabi/RestoreElectrum.md:77]:
```suggestion
- Copy your `Extended Account zprv`
```

File: docs/using-wasabi/RestoreElectrum.md [hunk 26]
```diff
@@ -1,107 +1,89 @@
...
+	- Copy your `Extended Account zpub`
```
Comment (RiccardoMasutti) [inline docs/using-wasabi/RestoreElectrum.md:78]:
```suggestion
- Copy your `Extended Account zpub`
```

File: docs/using-wasabi/RestoreElectrum.md [hunk 27]
```diff
@@ -1,107 +1,89 @@
...
+	:::
```
Comment (RiccardoMasutti) [inline docs/using-wasabi/RestoreElectrum.md:79]:
```suggestion
:::
```

File: docs/using-wasabi/RestoreElectrum.md [hunk 28]
```diff
@@ -1,107 +1,89 @@
...
+
+4. Open the wallet file you created in Electrum.
+
+5. Increase the gap limits by opening Electrum's `Console` and executing the following commands:
+
+	```
```
Comment (RiccardoMasutti) [inline docs/using-wasabi/RestoreElectrum.md:85]:
```suggestion
```
```

File: docs/using-wasabi/RestoreElectrum.md [hunk 29]
```diff
@@ -1,107 +1,89 @@
...
+	wallet.change_gap_limit(100)
```
Comment (RiccardoMasutti) [inline docs/using-wasabi/RestoreElectrum.md:86]:
```suggestion
wallet.change_gap_limit(100)
```

File: docs/using-wasabi/RestoreElectrum.md [hunk 30]
```diff
@@ -1,107 +1,89 @@
...
+	wallet.gap_limit_for_change = 100
```
Comment (RiccardoMasutti) [inline docs/using-wasabi/RestoreElectrum.md:87]:
```suggestion
wallet.gap_limit_for_change = 100
```

File: docs/using-wasabi/RestoreElectrum.md [hunk 31]
```diff
@@ -1,107 +1,89 @@
...
+	wallet.synchronize()
```
Comment (RiccardoMasutti) [inline docs/using-wasabi/RestoreElectrum.md:88]:
```suggestion
wallet.synchronize()
```

File: docs/using-wasabi/RestoreElectrum.md [hunk 32]
```diff
@@ -1,107 +1,89 @@
...
+	```
```
Comment (RiccardoMasutti) [inline docs/using-wasabi/RestoreElectrum.md:89]:
```suggestion
```
```

File: docs/using-wasabi/RestoreElectrum.md (hunk 2 above)
Comment (MaxHillebrand) [reply docs/using-wasabi/RestoreElectrum.md:34]: NACK - again, this is to align the tip box with the numbered list...

File: docs/using-wasabi/RestoreElectrum.md [hunk 33]
```diff
@@ -1,107 +1,89 @@
...
 }
 ---
 
-# Restoring Wasabi Wallet to Electrum
+# Restoring Wasabi Wallet in Electrum
 
-:::danger Bad for your privacy!
-If you do this, you send all your Wasabi addresses to 3rd party Electrum servers, losing the anonymity against those entities, so you must make a judgement call by being aware of this.
+:::danger Potential privacy leak!
+If you do not run a full node and your own Electrum server, you will leak all your addresses to random third party servers, losing anonymity against those entities, so you must make a judgement call by being aware of this.
```
Comment (yahiheb) [inline docs/using-wasabi/RestoreElectrum.md:11]:
```suggestion
If you do not run your own Electrum server, you will leak all your addresses to random third party servers, losing anonymity against those entities, so you must make a judgement call by being aware of this.
```

File: docs/using-wasabi/RestoreElectrum.md [hunk 34]
```diff
@@ -1,107 +1,89 @@
...
+	:::
 
-:::tip
-Steps to find your master private key inside Wasabi:
-- Go to `Advanced` tab -> `Wallet Info`.
-- Copy your `Extended Account zprv`
-:::
-
-The following steps are only necessary if you have previously chosen the option `I already have a seed`.
+	The following steps are only necessary if you have previously chosen the option `I already have a seed`.
```
Comment (yahiheb) [inline docs/using-wasabi/RestoreElectrum.md:39]:
```suggestion
The following steps are only necessary if you have previously chosen the option `I already have a seed`.
```

File: docs/using-wasabi/RestoreElectrum.md [hunk 35]
```diff
@@ -1,107 +1,89 @@
...
 
-5. Type your seed or paste your master private key
+5. Type your seed or paste your master private key.
 
-:::danger You are at risk!
-By typing the seed or pasting the master private key on a hot or compromised device, you risk losing your funds.
-:::
+	:::tip Get your master private key inside Wasabi:
+	- Go to `Advanced` tab -> `Wallet Info`.
+	- Copy your `Extended Account zprv`
```
Comment (yahiheb) [inline docs/using-wasabi/RestoreElectrum.md:36]:
```suggestion
	- Copy your `Extended Account zprv`.
```

File: docs/using-wasabi/RestoreElectrum.md [hunk 36]
```diff
@@ -1,107 +1,89 @@
...
+	    "use_encryption": false,
+	    "wallet_type": "standard"
+	}
+	```
+
+3. Replace `xprv` and `xpub` sections with your Wasabi wallet's `Extended Account zprv` and `Extended Account zpub`.
+
+	:::tip Get your master private and public keys inside Wasabi:
+	- Go to `Advanced` tab -> `Wallet Info`.
+	- Copy your `Extended Account zprv`
```
Comment (yahiheb) [inline docs/using-wasabi/RestoreElectrum.md:77]:
```suggestion
	- Copy your `Extended Account zprv`.
```

File: docs/using-wasabi/RestoreElectrum.md [hunk 37]
```diff
@@ -1,107 +1,89 @@
...
+	- Copy your `Extended Account zpub`
```
Comment (yahiheb) [inline docs/using-wasabi/RestoreElectrum.md:78]:
```suggestion
	- Copy your `Extended Account zpub`.
```

**Pull request merged**